package.json

```diff
@@ -30,20 +30,20 @@
     "node": ">=14"
   },
   "dependencies": {
+    "@types/request": "^2.48.8",
```

Comment:
Why is this still needed?

Reply:
In case someone still wants to use `requests` (or similar custom interface) we still support its types in index.d.ts.